src/subscript/config_jobs/PRTVOL2CSV

```diff
@@ -5,14 +5,16 @@ DEFAULT <DIR> share/results/volumes
 DEFAULT <OUTPUTFILENAME> simulator_volume_fipnum.csv
 
 DEFAULT <REGIONS> ""
+DEFAULT <FIPNAME> "FIPNUM"
 
 ARG_TYPE 0 STRING
 ARG_TYPE 1 STRING
 ARG_TYPE 2 STRING
 ARG_TYPE 3 STRING
 ARG_TYPE 4 STRING
+ARG_TYPE 5 STRING
 
-ARGLIST "--verbose" "--dir" <DIR> "--outputfilename" <OUTPUTFILENAME> "--yaml" <REGIONS> <DATAFILE>
+ARGLIST "--verbose" "--dir" <DIR> "--outputfilename" <OUTPUTFILENAME> "--yaml" <REGIONS> "--fipname" <FIPNAME> "--rename4webviz" <DATAFILE>
```

Comment:
In ARGLIST, should` --rename4webviz` option have a bool argument here defaulted to FALSE?

Can rename4webiz name be a bit misleading as it is only for on-premise webviz?
Could `rename2fipnum` be an alternative?

Reply:
Yes, changed name for the arguement now: ``rename2fipnum``